skin-schema.json

```diff
@@ -716,6 +716,36 @@
         }
       }
     },
+    "isVrAnimationEnabled": {
+      "description": "Shows if vr inform animation is enabled",
```

Comment:
Can we describe the animation that this setting controls? From reading this it's not clear to the user what will happen if this is enabled

Reply:
\+